droidlet/memory/sql_memory.py

```diff
@@ -609,6 +620,26 @@ def get_most_recent_incoming_chat(self, after=-1) -> Optional["ChatNode"]:
         else:
             return None
 
+    ###################
+    ## Logical form ###
+    ###################
+
+    def add_logical_form(self, logical_form: dict):
+        """Create a new ProgramNode
+
+        Args:
+            logical_form: the semantic parser's output
+        """
+        return ProgramNode.create(self, logical_form)
+
+    def get_logical_form_by_id(self, memid: str) -> "ProgramNode":
```

Comment:
this also... new search syntax will make these extraneous

Reply:
same as above ^

Reply:
also fine- commenting to warn rather than asking for change